src/element-internals.ts

```diff
@@ -403,6 +405,7 @@ export function forceCustomStateSetPolyfill(
  * polyfill as well.
  */
 export function forceElementInternalsPolyfill(forceCustomStateSet = true) {
+  let attachedFlag = false;
```

Comment:
Can you add a comment here about what this is for? I definitely get it from context, but the comments are for the next person who visits this file that isn't us.